src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin-top: 0.45em;
+  margin-bottom: 0.4em;
+}
+
+h2 {
+  font-size: 15px;
+  font-weight: normal;
+  margin-top: 1em;
+  margin-bottom: 1em;
```

Comment:
You can write `margin` in one line.

Reply:
Of course I can...
`margin: 1em X;`
  But is the X = 0? Or must it have some other value for passing tests? There is no answer in the task, but only the text: "set margin top and bottom" and nothing about left or right... did I do that point? Yes... And I did pass the test with default X. Must i try to pass the test with value, that is not mentioned in the task and I must search for what it is? I have many tasks for learning and I don't have time for this... Moreover, there has been a similar trouble... You were interested in it in your next comment.
P. S. Such comments don't help me, but confuse. Could you, please, to write like "it is bad, because ...", or "it's bad practice..."?.. Can you comment on the reason for such changes?

Reply:
Так, у даному випадку можна було покласти X = 0 (дефолтні горизонтальні межі нульові) — зазвичай пишуть саме так (щоб максимально уникнути дублювань).

Твій коментар зрозумілий, але рев’юверу важко завжди відразу ж давати стовідсотково конструктивний для студента фідбек. Істина народжується в дискусії.